src/main/java/org/tron/core/config/Parameter.java

```diff
@@ -104,11 +106,22 @@
 //    EXCHANGE_BALANCE_LIMIT,
   }
 
+  @Deprecated
   interface ForkBlockVersionConsts {
 
     int START_NEW_TRANSACTION = 4;
     int ENERGY_LIMIT = 5;
-    int VERSION_3_2_2 = 6;
+  }
+
+  enum ForkBlockVersionEnum {
+    VERSION_3_2_2(6);
+
+    @Getter
+    int value;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "value". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1849&issues=AWeszya0NP9b202rHNTT&open=AWeszya0NP9b202rHNTT)

[ISSUE_KEY]: AWeszya0NP9b202rHNTT